constructor.js

```diff
@@ -0,0 +1,147 @@
+var allMonths = [];
+
+var january = new Month('january');
+var february = new Month('february');
+var march = new Month('march');
+var april = new Month('april');
+var may = new Month('may');
+var june = new Month('june');
+var july = new Month('july');
+var august = new Month('august');
+var september = new Month('september');
+var october = new Month('october');
+var november = new Month('november');
+var december = new Month('december');
+
+function startUpCheckStorage() {
+
+  var newBill = [];
+  if (localStorage.storedBills) {
+    var parsedBills = JSON.parse(localStorage.getItem('storedBills'));
+    for (var i = 0; i < parsedBills.length; i++){
+			var currentBill = new BillObject(parsedBills[i].billMonthName, parsedBills[i].billAmount, parsedBills[i].billFrequency, parsedBills[i].billName);
+			newBill.push(currentBill);
+			// new BillObject(each.billMonthName, each.billAmount, each.billFrequency, each.billName)
+    }
+
+    for (var j = 0; j < newBill.length; j++) {
+			// console.log(newBill);
+			// console.log(item);
+      newBill[j].findAndUpdateMonth();
+    }
+  }
+  if (localStorage.storedBills) {
+
+
+    for (var i = 0; i < allMonths.length; i++) {
+      for (var j = 0; j < allMonths[i].billNameArray.length; j++) {
+        // console.log(allMonths[i]);
+        // console.log(allMonths[i].billObjectArray);
+        var currentObject = allMonths[i].billObjectArray[j];
+				var billSelector = document.getElementById('selectedBill')
+      	var billOption = document.createElement('option');
+				console.log(document.getElementById('selectedBill'));
+				console.log(billOption);
+      	billOption.value = currentObject.billName;
+      	billOption.textContent = currentObject.billName + " - " + currentObject.billMonthName;
+      	billOption.id = currentObject.billName;
+      	billSelector.appendChild(billOption);
+
+
+        var currentBillsList = document.getElementById("billList");
+        var newListItem = document.createElement("li");
+        newListItem.textContent = currentObject.billName + " $" + currentObject.billAmount + " - " + currentObject.billMonthName;
+        newListItem.id = currentObject.billName + '-li';
+        currentBillsList.appendChild(newListItem);
+      }
+    }
+  }
+}
+
+
+function Month(name){
+
+  this.monthName = name;
+  this.billObjectArray = [];
+  this.billNameArray = [];
+  this.billAmountArray = [];
+  this.roommateNameArray = [];
+  this.grandTotal = 0;
+  allMonths.push(this);   //push newly instantiated billobject to array in tracker
+}
+
+Month.prototype.addAllBills = function() {
+
+};
+
+var helperFunctions = {
```

Comment:
This is creative, and a great way to isolate your helper functionality into a single space.